.github/workflows/test-suite-web-e2e-pw.yml

```diff
@@ -144,21 +136,18 @@ jobs:
           npx playwright install --with-deps chromium
           docker compose pull ${{ matrix.CONTAINERS }}
 
-      - name: Run Playwright e2e tests
+      - name: Run Playwright e2e web tests
         env:
           COMPOSE_FILE: ./docker/docker-compose.suite-ci-pw.yml
           BASE_URL: https://dev.suite.sldev.cz/suite-web/${{ steps.extract_branch.outputs.branch }}/web/
-          ## Keep retries enabled once stabilized
-          ALLOW_RETRY: true
-          TEST_GROUP: ${{ matrix.TEST_GROUP }}
           GITHUB_ACTION: true
           CURRENTS_PROJECT_ID: Og0NOQ
           CURRENTS_RECORD_KEY: ${{ secrets.CURRENTS_RECORD_KEY }}
           CURRENTS_CI_BUILD_ID: pr-run-${{github.run_id}}
         run: |
           docker compose up -d ${{ matrix.CONTAINERS }}
-          echo "Starting Playwright Web test group ${TEST_GROUP}"
-          yarn workspace @trezor/suite-desktop-core test:e2e:web --grep=${TEST_GROUP}
+          echo "Starting Playwright Web test group ${{ matrix.TEST_GROUP }}"
```

Comment:
Unify how vars are passed to bash and adds safety net --forbid-only. Playwright docs: Whether to disallow test.only. Useful on CI.